install.sh

```diff
@@ -178,11 +178,11 @@ relocate_python3() {
     local pythonpath="$(dirname "$pythoncmd")"
 
     if [ ! -x "$script" ]; then
-        install -m755 "$script" "$install"
+        cp "$script" "$install"
```

Comment:
IMHO, `install` is the more idiomatic way to install files on GNU/Linux. as it is designed to address the problems one could be facing when using the combo of `cp` and `chmod`.

i understand that the pain is that we are using `cp -r`  in this script. but probably the better way is to change `cp -r` to  `cp -pr`  as suggested by https://docs.fedoraproject.org/en-US/packaging-guidelines/#_timestamps ?  as the functionalities of `install.sh` overlaps with .spec in rpm packaging, the difference is that we don't have the packaging guideline and rpm macros in our arsenal.